Server/DeviceManagement/DeviceManagement.Domain/UserAggregate/Events/UserCreatedEvent.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System;
+using ModularHouse.Libraries.InternalMessaging.DomainEvents.Abstractions;
+
+namespace ModularHouse.Server.DeviceManagement.Domain.UserAggregate.Events;
+
+public record UserCreatedEvent(User User, Guid TransactionId) : IDomainEvent;
```

Comment:
I'm not sure, if it's good idea to send domain data like `User` by `DomainEvent` to upper layers;
I think that it makes sense to send not `User`, but `UserDto` or `Guid UserId`
Let me know what you think about this too =)